pset-functions.js

```diff
@@ -31,6 +31,28 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack (playerCardScore, dealerCardScore){
```

Comment:
@rvaswani85 good job, the logic is good.., however, your return values from within the function should just be the value itself and not a string like "Result: 21". See [this implementation](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-functions.js#L33) for more details